cli/src/extension_api.ts

```diff
@@ -181,6 +181,28 @@ export class PhylumApi {
         return Deno.core.opAsync('get_projects', group);
     }
 
+    /// Create a project.
+    ///
+    /// # Returns
+    /// 
+    /// The id of the newly created project:
+    ///
+    /// ```
+    /// "5d6eaa97-dff8-dead-a619-bcafffefeef0"
+    /// ```
+    static async createProject(name: string, group?: string): string {
+        return Deno.core.opAsync('create_project', name, group);
+    }
+
+    /// Deletes a project.
```

Comment:
```suggestion
    /// Delete a project.
```

Just for consistency, should either be creates/deletes or create/delete.